### PR TITLE
DOC: add examples to build_contiguity and build_triangulation

### PR DESCRIPTION
**Pull request by lisawink**
Add docstring examples to build_contiguity and build_triangulation

### CODE REVIEW

**Review by jGaboardi — APPROVED** [review]

File: libpysal/graph/base.py [hunk 1]
```diff
@@ -840,6 +890,43 @@ def build_triangulation(
...
+                       BoroCode  ...                                           geometry
+        BoroName                 ...
+        Staten Island         5  ...  MULTIPOLYGON (((970217.022 145643.332, 970227....
+        Queens                4  ...  MULTIPOLYGON (((1029606.077 156073.814, 102957...
+        Brooklyn              3  ...  MULTIPOLYGON (((1021176.479 151374.797, 102100...
+        Manhattan             1  ...  MULTIPOLYGON (((981219.056 188655.316, 980940....
+        Bronx                 2  ...  MULTIPOLYGON (((1012821.806 229228.265, 101278...
+        [5 rows x 4 columns]
+
+        Note that the function requires point geometry as an input
```
Comment (martinfleis) [inline libpysal/graph/base.py:910]:
```suggestion
        Note that the method requires point geometry (or an array of coordinates 
        representing points) as an input.
```

File: libpysal/graph/base.py [hunk 2]
```diff
@@ -403,6 +403,56 @@ def build_contiguity(cls, geometry, rook=True, by_perimeter=False, strict=False)
         -------
         Graph
             libpysal.graph.Graph encoding contiguity weights
+
+        Example
+        -------
```
Comment (martinfleis) [inline libpysal/graph/base.py:408]:
```suggestion
        Examples
        --------
```
Needs to be plural to be understood by sphinx (docs builder).

File: libpysal/graph/base.py [hunk 3]
```diff
@@ -840,6 +890,43 @@ def build_triangulation(
         -------
         Graph
             libpysal.graph.Graph encoding triangulation weights
+
+        Example
+        -------
```
Comment (martinfleis) [inline libpysal/graph/base.py:895]:
```suggestion
        Examples
        --------
```

**Review by martinfleis — COMMENTED** [review]
A few notes.

**Comment by martinfleis** [comment]
pre-commit.ci autofix

**Review by martinfleis — APPROVED** [review]

**Pull request merged**